Common/src/main/java/com/yungnickyoung/minecraft/betterendisland/mixin/EndDragonFightMixin.java

```diff
@@ -583,10 +606,10 @@ public void betterendisland_setDragonKilled(EnderDragon dragon, CallbackInfo ci)
         if (dragon.getUUID().equals(this.dragonUUID)) {
             this.dragonEvent.setProgress(0.0F);
             this.dragonEvent.setVisible(false);
-            this.betterendisland$spawnPortal(true, true);
+            this.betterendisland$spawnPortal(true, true, BetterEndIslandCommon.CONFIG.regenerateTowerOnDragonDeath);
             level.explode(null, this.portalLocation.getX(), this.portalLocation.getY(), this.portalLocation.getZ(), 6.0F, Level.ExplosionInteraction.NONE);
             this.spawnNewGateway();
-            if (!this.previouslyKilled || BetterEndIslandCommon.moreDragonEggs || BetterEndIslandCommon.CONFIG.resummonedDragonDropsEgg) {
```

Comment:
How come the check for `BetterEndIslandCommon.CONFIG.resummonedDragonDropsEgg` was removed?

Reply:
Ah, that was an inadverent change. Will add back.